src/styles/blocks.scss/card.scss

```diff
@@ -0,0 +1,105 @@
+.card {
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+  width: 200px;
+  padding: 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition: transform 300ms;
+
+  &__product-image {
+    padding: 16px 3px 0;
+    display: flex;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__product-name {
+    margin: 0;
+    padding-top: 40px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
+  }
+
+  &__product-code {
+    margin: 0;
+    padding-top: 4px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+  }
+
+  &__product-rating {
+    padding-top: 16px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+
+    &-review {
+      margin: 0;
+      display: flex;
+      font-weight: 400;
+      font-size: 10px;
+      line-height: 14px;
+      color: #060b35;
+      align-self: flex-end;
+    }
+  }
+
+  &__product-price {
+    padding-top: 24px;
+    padding-bottom: 16px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    line-height: 18px;
+
+    &-text {
+      margin: 0;
+      font-weight: 400;
+      font-size: 12px;
+      color: #616070;
+    }
+
+    &-numbers {
+      margin: 0;
+      font-weight: 700;
+      font-size: 16px;
+    }
+  }
+
+  &__product-buy-link {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border-radius: 5px;
+    width: 100%;
+    height: 40px;
+    background-color: $main-color;
```

Comment:
You could use a variable for the background color here. This would make your code more maintainable and easier to read. For example, you could define a $button-bg-color variable in your variables.scss file and use it here.